persistence-model/src/test/scala/hmda/persistence/serialization/parser/HmdaFileParserProtobufSerializerSpec.scala

```diff
@@ -0,0 +1,47 @@
+package hmda.persistence.serialization.parser
+
+import hmda.model.fi.lar.LarGenerators
+import hmda.model.fi.ts.TsGenerators
+import hmda.model.parser.LarParsingError
+import hmda.persistence.messages.events.processing.HmdaFileParserEvents._
+import org.scalacheck.Gen
+import org.scalatest.{ MustMatchers, PropSpec }
+import org.scalatest.prop.PropertyChecks
+
+class HmdaFileParserProtobufSerializerSpec extends PropSpec with PropertyChecks
+    with MustMatchers with LarGenerators with TsGenerators {
+
+  val serializer = new HmdaFileParserProtobufSerializer()
+
+  def listOfStringsGen: Gen[List[String]] = Gen.listOf(Gen.alphaStr)
+  def intGen: Gen[Int] = Gen.choose(0, 100000)
```

Comment:
Same comment as above, I don't think you need to explicitly write the types here